isaaclab.bat

```diff
@@ -244,12 +241,6 @@ if "%arg%"=="-i" (
         set ext_folder="%%d"
         call :install_isaaclab_extension
     )
-    call !python_exe! -m pip show isaacsim-rl > nul 2>&1
```

Comment:
same changes should be made for --install option

Reply:
Done.